var/spack/repos/builtin/packages/netcdf-cxx4/package.py

```diff
@@ -56,6 +56,12 @@ def libs(self):
     def configure_args(self):
         config_args = []
 
+        if self.spec.satisfies('^mpi'):
```

Comment:
I would specify the condition like this (like in #16719):
```python
netcdf_c_spec = self.spec['netcdf-c']
if '+mpi' in netcdf_c_spec or '+parallel-netcdf' in netcdf_c_spec:
```
However, the current solution is also fine in my opinion.

Reply:
When netcdf-c is updated (For example new variant is added and the variant is needed MPI),
#16719 solution is needed to change netcdf-cxx.
But thi PR is not needed to change.

Reply:
I see.
I Changed to look at variant of netcdf-c.